cou/utils/openstack.py

```diff
@@ -71,6 +71,28 @@
     "octavia",
 ]
 
+SUBORDINATES = [
+    "barbican-vault",
+    "ceilometer-agent",
+    "cinder-backup-swift-proxy",
+    "cinder-ceph",
+    "cinder-lvm",
+    "cinder-netapp",
+    "cinder-purestorage",
+    "keystone-kerberos",
+    "keystone-ldap",
+    "keystone-saml-mellon",
+    "magnum-dashboard",
+    "manila-dashboard",
+    "manila-generic",
+    "masakari-monitors",
+    "neutron-api-plugin-ironic",
+    "neutron-api-plugin-ovn",
```

Comment:
I think [neutron-api-plugin-arista](https://docs.openstack.org/charm-guide/latest/project/charm-delivery.html) can be also included here

Reply:
added